pkg/cmd/grafana-cli/commands/reset_password_command.go

```diff
@@ -16,7 +19,20 @@ import (
 const AdminUserId = 1
 
 func resetPasswordCommand(c utils.CommandLine, sqlStore *sqlstore.SqlStore) error {
-	newPassword := c.Args().First()
+	newPassword := ""
+
+	if c.Bool("password-from-stdin") {
+		logger.Infof("\nNew Password: ")
+
+		reader := bufio.NewReader(os.Stdin)
+		line, err := reader.ReadString('\n')
+		if err != nil {
+			return fmt.Errorf("can't read password from stdin: %w", err)
+		}
+		newPassword = strings.TrimSuffix(line, "\n")
```

Comment:
Using bufio.Scanner should be a cleaner solution:
```suggestion
		scanner := bufio.NewScanner(os.Stdin)
		if ok := scanner.Scan(); !ok {
			return fmt.Errorf("can't read password from stdin")		    
		}
		if err := scanner.Err(); err != nil {
			return fmt.Errorf("can't read password from stdin: %w", err)
		}
		newPassword = scanner.Text()
```

Reply:
That's the best way to read from buffers. I have no objections. Let's switch to this solution.